document-portal/file-transfer.c

```diff
@@ -281,6 +284,7 @@ file_transfer_execute (FileTransfer *transfer,
   fds = g_new (int, n_fds);
   parent_devs = g_new (int, n_fds);
   parent_inos = g_new (int, n_fds);
+  the_flags = g_new (guint32, n_fds);
```

Comment:
`guint32` → `uint32_t`

Reply:
`DocumentAddFullFlags` is the actual type. Will use that instead.